README.md

```diff
@@ -1,2 +1,74 @@
-# tw-academy-s9
-Homework for the TW Academy
+# Sample Style Guide
+## Technical Writer Academy - May 2021
+---
+## Table of contents
+
+[Bold](#Bold)  
+[Italics](#Italics)  
+[Fixed Font](#Fixed-font)  
+[Abbreviations](#Abbreviations)  
+[Capitalization](#Capitalization)  
+[Acronyms](#Acronyms)  
+[Commas in Lists](#Commas-in-Lists)  
+[Numbers](#Numbers)  
+[Contractions](#Contractions)  
+[Apostrophes](#Apostrophes)   
+## Bold  
+Use bold to highlight button names, fields names, menus, radio buttons, or checkboxes. For example:  
+- Click the **Create** button to start the process.  
+- Enter your IP address in the **IP Address** field.  
+
+## Italics
+Use italics to emphasize a word or group of words. For example:
+- At least two reviews are required before you check in your code.
+- You must not enter without your ID card.  
+> **Important:** Use this style sparingly to avoid diluting it's strenght.  
+  
+  ## Fixed font  
```

Comment:
There is some unnecessary indentation here. It doesn't seem to be causing problems with the rendered markdown, but it does prevent the code from highlighting properly.